api/v1/views/__init__.py

```diff
@@ -0,0 +1,3 @@
+from flask import Blueprint
+app_views = Blueprint('app_views', __name__)
```

Comment:
Url prefix not asigned and  must be at  /api/v1

Reply:
I think that while in the _Blueprint_ instance and in the current part you can send this parameter, for the time being let's test if it works with this current configuration. What do you think?

Reply:
Ok, Let's move forward